app/src/main/java/ar/com/wolox/android/example/ui/newsDetail/NewsDetailFragment.kt

```diff
@@ -0,0 +1,131 @@
+package ar.com.wolox.android.example.ui.newsDetail
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import android.widget.Toast
+import android.widget.ToggleButton
+import androidx.appcompat.widget.Toolbar
+import androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.model.News
+import ar.com.wolox.android.example.ui.newsDetail.ImageFullScreenActivity.Companion.IMAGE_URL
+import ar.com.wolox.wolmo.core.fragment.WolmoDialogFragment
+import com.facebook.drawee.view.SimpleDraweeView
+import kotlinx.android.synthetic.main.fragment_news.*
+import java.io.Serializable
+import javax.inject.Inject
+
+class NewsDetailFragment @Inject constructor() : WolmoDialogFragment<NewsDetailPresenter>(), INewsDetailView {
+
+    private lateinit var vNewsDetailPicture: SimpleDraweeView
```

Comment:
Hey! What happened here? If you're using Kotlin you can import the view directly, there's no need to declare it.